Dockerfile

```diff
@@ -31,12 +31,16 @@ ENV GOPATH=/root/.go \
 
 WORKDIR /home/deepfence/src/SecretScanner
 COPY . .
-RUN go build -v -i \
-    && mv /home/deepfence/src/SecretScanner/SecretScanner /home/deepfence/src/SecretScanner/config.yaml /tmp \
-    && rm -rf /home/deepfence/src/SecretScanner/* \
-    && mv /tmp/SecretScanner /tmp/config.yaml /home/deepfence/src/SecretScanner \
-    && rm -rf /usr/local/go-1.17.5
+RUN go build -v -i
+
+FROM ubuntu:focal
```

Comment:
Can we use alpine ?

Reply:
We can use alpine with a few tweaks. Bad news is, the size reduction is not that good because we need to bring in docker. It is still lower than current size: ~280 MB